integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -1980,4 +1980,28 @@ public static ExecResult kubectlpatch(String domainUid, String domainNS, String
     return exec(cmd, true);
   }
 
+  /**
+   * create configmap in the given namespace. Also creates label.
+   * @param cmName
+   * @param fileOrDirPath
+   * @param namespace
+   * @throws Exception
+   */
+  public static void createConfigMap(String cmName, String fileOrDirPath,
```

Comment:
What will be the content of this config map?

Reply:
@vanajamukkara ?

Reply:
this will load the file or files from the given input